content/blogs/control-vmware-with-kestra.md

```diff
@@ -0,0 +1,147 @@
+
+### Trigger Capabilities
+
+The plugin supports triggering workflows based on a wide range of VMware events, allowing for highly responsive and automated infrastructure operations. Here are the supported events:
+
+- **VM Created** — Triggered when a virtual machine is created  
+- **VM Deployed** — Triggered upon successful deployment of a VM  
+- **VM Cloned** — Captures the cloning of a virtual machine  
+- **VM Renamed** — Activated when a VM is renamed  
+- **VM Removed** — Triggered when a VM is deleted or removed  
+- **VM Powered On** — Initiated when a VM starts  
+- **VM Powered Off** — Triggered when a VM is shut down  
+- **VM Suspended** — Detects suspension of a VM  
+- **VM Failed to Power On** — Alerts when a power-on attempt fails  
+- **VM Failed to Power Off** — Alerts when a shutdown attempt fails  
+- **VM Failed to Suspend** — Indicates an error during suspension  
+- **VM Failed to Reset** — Captures failed attempts to reset a VM  
+
+These events can trigger downstream workflows such as [sending alerts](https://kestra.io/docs/how-to-guides/alerting), updating configuration databases, or kicking off CI pipelines. For example, you can automatically notify a Slack channel when a new VM is created or start a backup workflow when a VM is powered off.
+
+```yaml
+id: vm-event-listen
+namespace: your.company
+
+triggers:
+  - id: watch
+    type: io.kestra.plugin.ee.vmware.esxi.Trigger
+    interval: PT1M
+    server: "exsi.vsphere.local"
+    port: 443
+    username: "foo"
+    password: "bar"
+    eventType: VM_POWERED_OFF
+    vmNameRegex: "vm-.*"
+
+tasks:
+  - id: foreach-event
+    type: io.kestra.plugin.core.flow.ForEach
+    values: "{{ trigger.events }}"
+    tasks:
+      - id: notify-slack
+        type: io.kestra.plugin.notifications.slack.SlackIncomingWebhook
+        url: "{{ secret('SLACK_WEBHOOK') }}"
+        messageText: "A new VM named *{{ json(taskrun.value).vmName }}* was created at {{ json(taskrun.value).timestamp }}."
+```
+
+This turns your Kestra workflows into **responsive infrastructure agents**, not just scheduled jobs.
+
+---
+
+## A Smarter Way to Orchestrate VMware
+
+Virtual machines are still the foundation of enterprise infrastructure—but managing them shouldn’t be stuck in the past. The VMware plugin for Kestra EE brings first-class orchestration to your virtualization stack, making it easy to:
+
+- Automate repetitive VM operations like provisioning, powering on/off, and cloning
+- Coordinate lifecycle management across environments in a version-controlled, declarative way
+- React to real-time changes and trigger downstream workflows without any manual intervention
+
+Infrastructure automation is about: speed, resilience, traceability, and making your platform team more effective. This plugin helps you to get there.
+
+**Start using the VMware plugin in Kestra EE and bring your virtual environments into the modern orchestration era.**
```

Comment:
It would be good to add the link to the plugin somewhere at the bottom: https://kestra.io/plugins/plugin-ee-vmware